draft-ietf-taps-transport-security.md

```diff
@@ -482,30 +485,33 @@ but this document considers them together, since that is the most common pattern
 
 #### IKEv2
 
-IKEv2 is a control protocol that runs on UDP port 500. Its primary goal is to generate keys for Security Associations (SAs).
-An SA contains shared (cryptographic) information used for establishing other SAs or keying ESP; See {{ESP}}.
-IKEv2 first uses a Diffie-Hellman key exchange to generate keys for the "IKE SA", which is a set of keys
-used to encrypt further IKEv2 messages. It then goes through a phase of authentication in which both peers
-present blobs signed by a shared secret or private key, after which another set of keys is derived, referred
-to as the "Child SA". These Child SA keys are used by ESP.
+IKEv2 is a control protocol that runs on UDP port 500 or TCP port 4500. (IKEv2 cannot run over
+UDP on any other port.) Its primary goal is to generate keys for Security Associations (SAs).
+An SA contains shared (cryptographic) information used for establishing other SAs or keying ESP;
+See {{ESP}}. IKEv2 first uses a Diffie-Hellman key exchange to generate keys for the "IKE SA",
+which is a set of keys used to encrypt further IKEv2 messages. IKE then performs a phase of
+authentication in which both peers present blobs signed by a shared secret or private key that
+authenticates the entire IKE exchange and the IKE identities. IKE then derives further sets of
+keys on demand, which together with traffic policies are referred to as the "Child SA". These
+Child SA keys are used by ESP.
 
 IKEv2 negotiates which protocols are acceptable to each peer for both the IKE and Child SAs using
-"Proposals". Each proposal may contain an encryption algorithm, an authentication algorithm, a
-Diffie-Hellman group, and (for IKE SAs only) a pseudorandom function algorithm. Each peer may
+"Proposals". Each proposal contains an encryption with authentication algorithm or an AEAD algorithm,
+a Diffie-Hellman group, and (for IKE SAs only) a pseudorandom function algorithm. Each peer may
 support multiple proposals, and the most preferred mutually supported proposal is chosen during
 the handshake.
 
 The authentication phase of IKEv2 may use Shared Secrets, Certificates, Digital Signatures, or an
 EAP (Extensible Authentication Protocol) method. At a minimum, IKEv2 takes two round trips to set
 up both an IKE SA and a Child SA. If EAP is used, this exchange may be expanded.
 
-Any SA used by IKEv2 can be rekeyed upon expiration, which is usually based either on time or
+Any SA used by IKEv2 can be rekeyed before expiration, which is usually based either on time or
 number of bytes encrypted.
 
 There is an extension to IKEv2 that allows session resumption {{RFC5723}}.
 
 MOBIKE is a Mobility and Multihoming extension to IKEv2 that allows a set of Security Associations
-to migrate over different addresses and interfaces {{RFC4555}}.
+to migrate over different outer IP addresses and interfaces {{RFC4555}}.
```

Comment:
Should we assume the user knows what an "outer IP address" is? I know there's an informational reference there, but "outer IP address" seems like the answer to a question the user hasn't asked.